rq/job.py

```diff
@@ -750,4 +748,58 @@ def register_dependency(self, pipeline=None):
             connection.sadd(dependents_key, self.id)
             connection.sadd(self.dependencies_key, dependency_id)
 
+    @property
+    def dependencies_job_ids(self):
```

Comment:
Should this be `dependency_ids`?

Reply:
0672cd00c6ab5268f83887e66990726904cbe1f9